lib/image_normaliser.rb

```diff
@@ -0,0 +1,40 @@
+# frozen_string_literal: true
+
+require "mini_magick"
+
+class ImageNormaliser
+  attr_reader :issues, :raw_file
+
+  def initialize(raw_file)
+    @raw_file = raw_file
+  end
+
+  def normalise
+    @issues = Requirements::CheckerIssues.new
+    check_image_is_not_animated
+    check_image_is_big_enough
+    issues.any? ? return : normalised_image
```

Comment:
this seems a bit strange, maybe `normalised_image if issues.empty?`

Reply:
You and @1pretz1 clearly think alike :-). Fixed.